approval_frame/backends.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth.backends import ModelBackend
+from django.contrib.auth.models import User
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError, MultipleObjectsReturned
+
+class EmailOrUsernameBackend(ModelBackend):
+    """ Class to authenticate the user based on the given 
```

Comment:
Oh cool! I didn't know about the PEP convention documentation. Thanks for that, I can include it in the style guide. I am just following the [Django source code] (https://github.com/django/django/blob/master/django/contrib/auth/backends.py) style. It's still compatible with what the PEP docs say and it looks cleaner.